WooCommerce/Classes/ViewRelated/Dashboard/Analytics Hub/AnalyticsHubViewModel.swift

```diff
@@ -27,4 +28,56 @@ final class AnalyticsHubViewModel: ObservableObject {
                                                              trailingValue: "$57,99",
                                                              trailingDelta: "-16%",
                                                              trailingDeltaColor: .withColorStudio(.red, shade: .shade40))
+
+    // MARK: Private data
+
+    /// Order stats for the current selected time period
+    ///
+    @Published private var currentOrderStats: OrderStatsV4 = {
```

Comment:
What's the reason for these to be published properties? 🤔

Reply:
I thought if we make them published properties, we can listen for changes to the data (i.e. when a different time period is selected and new data is synced) to update what is displayed on the cards. WDYT?